conseil-common/src/test/scala/tech/cryptonomic/conseil/common/bitcoin/BitcoinFixtures.scala

```diff
@@ -229,4 +231,67 @@ trait BitcoinFixtures {
       vout = List(outputResult)
     )
   }
+
+  /**
+    * Fixtures containing Slick rows.
+    */
+  object DbFixtures {
```

Comment:
Are those used anywhere or just implemented for the future use?

Reply:
It's part of existing test spec in `BitcoinPersistenceTest`